notebook/services/contents/filemanager.py

```diff
@@ -275,16 +281,18 @@ def _dir_model(self, path, content=True):
                     continue
 
                 try:
-                    st = os.stat(os_path)
+                    st = f_stat(os_path)
                 except OSError as e:
                     # skip over broken symlinks in listing
                     if e.errno == errno.ENOENT:
                         self.log.warning("%s doesn't exist", os_path)
                     else:
-                        self.log.warning("Error stat-ing %s: %s", os_path, e)
+                        self.log.warning("Error stat-ing %s: %s", (os_path, e))
```

Comment:
Adding these parentheses will cause a formatting error:

    TypeError: not enough arguments for format string